azurelinuxagent/common/osutil/suse.py

```diff
@@ -27,16 +27,11 @@ class SUSE11OSUtil(DefaultOSUtil):
     def __init__(self):
         super(SUSE11OSUtil, self).__init__()
         self.jit_enabled = True
-        self.dhclient_name='dhcpcd'
+        self.dhclient_name = 'dhcpcd'
 
     def set_hostname(self, hostname):
         fileutil.write_file('/etc/HOSTNAME', hostname)
-        try: 
-            shellutil.run_command(["hostname", hostname])
-        except Exception as e:
-            raise OSUtilError(
-                "Failed to set hostname. Error: {}".format(ustr(e))
-            )
+        self._run_command_without_raising(["hostname", hostname], log_error=False)
```

Comment:
The exceptions were being raised as OSUtils before but you are not raising them anymore. You should use the `_run_command_raising_OSUtilError` instead

Reply:
No, originally no exceptions were being raised, we want to keep that behavior.

Reply:
Looks like we weren't raising anything before. Sorry for the confusion. Resolving comment